src/styles/blocks/nav.scss

```
@@ -0,0 +1,53 @@
+.nav {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  color: $primary-color;
+
+  &__list {
+    display: flex;
+
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: inline-block;
+    padding: 23px 0 19px;
+
+    text-decoration: none;
+    color: inherit;
+  }
+
+  &__link:visited {
+    color: $primary-color;
+  }
+
+  &__link:hover {
+    color: $highlight-color;
+    transition: color 0.2s;
+  }
```

implement this part inside &__link
```suggestion
  &__link {
    display: inline-block;
    padding: 23px 0 19px;

    text-decoration: none;
    color: inherit;
    
    &:visited {
      color: $primary-color;
  }

    &:hover {
      color: $highlight-color;
      ransition: color 0.2s;
    }
  }  
```

---

I agree. I'll make changes.